src/Elders.Cronus/Discoveries/DiscoveryScanner.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Elders.Cronus.Logging;
+
+namespace Elders.Cronus.Discoveries
+{
+    public class DiscoveryScanner : DiscoveryBasedOnExecutingDirAssemblies
+    {
+        private readonly static ILog log = LogProvider.GetLogger(typeof(DiscoveryScanner));
+        private readonly CronusServices cronusServices;
+
+        public DiscoveryScanner(CronusServices cronusServices)
+        {
+            this.cronusServices = cronusServices;
+        }
+
+        protected override DiscoveryResult DiscoverFromAssemblies(DiscoveryContext context)
+        {
```

Comment:
Null checks?

Reply:
I guess we are leaving it as is for the moment